pipeline-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -7,13 +7,14 @@ def branches = [:]
 //and jenkins will unite them into a single run of the job
 
 for (int i = 0; i < 4; i++) {
+  def _i = i //if we tried to use i below, it would equal 4 in each job execution.
```

Comment:
:+1: 

NIT: _ makes me a wee bit uneasy, how about `def index = i` instead?

Btw, as discussed in some other PR for this repo, those examples are designed to be exaggeratedly understandable. So I guess this would even greater to add a link to a more thorough documentation about why this is so with _Groovy_.